nmdc_schema/migrators/partials/migrator_from_11_0_3_to_11_1_0/migrator_from_11_0_3_to_11_1_0_part_3.py

```diff
@@ -0,0 +1,44 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+
+
+class Migrator(MigratorBase):
+    r"""
+    Migrates a database between two schemas.
+
+    This migrator removes the `has_peptide_quantificiations` field from all documents that represent an instance of the `MetaproteomicsAnalysis` class.
+    """
+    _from_version = "11.0.3"
```

Comment:
Since this is not the first partial migrator in the sequence, I suggest updating this to identify the _preceding_ partial migrator.

```suggestion
    _from_version = "11.1.0.part_2"
```

Reply:
I will go ahead and commit this change since it's after hours and the lead repo maintainer is in a timezone that is a few hours ahead of the other contributors (and so may start processing this PR before the other contributors have begun work for the day).

Y'all can undo this change if you want to, after that.